src/site/content/en/handbook/quick-start/index.md

```diff
@@ -25,9 +25,9 @@ If the piece you'd like to publish is time sensitive, make sure to submit the is
 1. The tech writer will review using the [web.dev content checklist](/handbook/content-checklist) and work with you to polish the piece.
 
 ## Publishing
-1. Once you and the tech writer think the piece is good to go, submit a pull request on GitHub with your content authored in Markdown. (Check out the "[web.dev markup](/handbook/#web.dev-markup)" section for details.)
+1. Once you and the tech writer think the piece is good to go, submit a pull request on GitHub with your content authored in Markdown. Check out the [web.dev markup](/handbook/#web.dev-markup) section for details.
```

Comment:
maybe "Save time during the review process and make sure your doc follows the web.dev markup guidelines."

Reply:
SGTM

Reply:
Fixed in Google Doc